build/Targets/Analyzers.Versions.targets

```diff
@@ -1,73 +1,73 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <PropertyGroup>
-      <MetaCompilationAnalyzersSemanticVersion>1.2.0</MetaCompilationAnalyzersSemanticVersion>
```

Comment:
If the semanticversion and the prereleaseversion is going to be the same for all components then we can have a 2 properties `SemanticVersion` and `PreReleaseVersion` and use the value of the properties in the changes below. The next time we need to bump the version, we only need to modify  `SemanticVersion` and `PreReleaseVersion` and save the pain of modifying everything